mmv1/products/sql/api.yaml

```diff
@@ -351,6 +351,12 @@ objects:
                 description: |
                     Whether the mysqld should default to 'REQUIRE X509' for
                     users connecting over IP.
+              - !ruby/object:Api::Type::Boolean
```

Comment:
I suspect this block isn't used because the resource is handwritten. Do you know if it is needed? It's possible this was converted to handwritten but the yaml file was not removed (if that's the case, apologies for the confusion, we need to delete some of these).

Reply:
I have reverted this unnecessary edit as well.